app/middleware/createGpViewModel.js

```diff
@@ -23,6 +24,10 @@ function getChoicesProfileLink(gpData) {
   return `http://www.nhs.uk/Services/GP/Overview/DefaultView.aspx?id=${gpData.choicesId}`;
 }
 
+function getChoicesEditLink(gpData) {
+  return `https://${liveServiceHostname}/Personalisation/Login.aspx?ReturnUrl=/Services/GP/Overview/DefaultView.aspx?id=${gpData.choicesId}`;
```

Comment:
Should this be `http` given the live site is `http` only?
Would it be easier to have the full url to the login page come from the config value? That way it could be changed to a different page without having to change the code.
This function feels like it would be better placed in its own file then if I need to change the login link I can go to the `loginLink.js` module rather than have to find the `createGpViewModel.js` module - just a thought.

Reply:
Agree that we should have the full URL in the env var.

It is purely a mechanism to facilitate testing in that we have no test GP logins in the live system to test that this works.